modules/compression/compression.c

```diff
@@ -158,6 +164,8 @@ static cmd_export_t cmds[]={
 	{"mc_compact",	  (cmd_function)mc_compact, {
 		{CMD_PARAM_STR|CMD_PARAM_OPT|CMD_PARAM_FIX_NULL,
 			fixup_whitelist_compact, fixup_whitelist_free},
+		{CMD_PARAM_STR|CMD_PARAM_OPT|CMD_PARAM_FIX_NULL,
```

Comment:
Please add the details on this new `n` flag into the documentation file ( modules/compression/doc/compression_admin.xml ).